action.yml

```diff
@@ -53,12 +55,25 @@ runs:
         INPUT_TAG: ${{ inputs.release_tag }}
       shell: bash
 
+    - id: determine_release_title
+      run: |
+        if [ -n "$INPUT_TITLE" ]; then
+          title=$INPUT_TITLE
```

Comment:
```suggestion
          title="$INPUT_TITLE"
```

Reply:
Addressed in [3bedb9f](https://github.com/cli/gh-extension-precompile/pull/45/commits/3bedb9f24dba40555797772d0a7cb539729c84a5).